lib/modules/dosomething/dosomething_home/dosomething_home.module

```
@@ -14,18 +14,15 @@ function dosomething_home_preprocess_node(&$vars) {
 
   // Replaces any global/site tokens in the title.
   $vars['title'] = token_replace($vars['title']);
+
   // Adds $subtitle variable.
   $vars['subtitle'] = $vars['field_subtitle'][0]['safe_value'];
+
   // Store this node as the signup source.
   $source = '?source=node/' . $vars['nid'];
 
+  // Prepare campaign thumbnails for homepage
   $campaigns = $vars['field_campaigns'];
-
-  if (empty($campaigns)) {
-    // Fallback to old stuff.
-    $campaigns = array(362, 832, 8, 843, 345, 31, 15, 17, 850);  
-  }
-
   foreach ($campaigns as $key => $campaign) {
```

Fair point. Should the fallback just be to create an empty array then? Falling back to arbitrary campaign NIDs seems wonky to me.